api_draft.txt

```diff
@@ -0,0 +1,128 @@
+
+################################################################################
+Arimaa Game Server JSON API
+################################################################################
+
+#TODO - see todos below
+
+All requests and responses are in JSON.
+
+Shared types:
+<Message> = {"message":<String>}
+<Error> = {"error":<String>}
+<GameID> = <Int>   #Identifier for a game
```

Comment:
Yeah that would be nice. Also this way we can prevent conflicts between our id and the old ids.